common/util/FileUtil.cpp

```diff
@@ -0,0 +1,36 @@
+#include "FileUtil.h"
+#include <iostream>
+#include <stdio.h>  /* defines FILENAME_MAX */
+
+#ifdef _WIN32
+  #include <direct.h>
+  #define GetCurrentDir _getcwd
+#else
+  #include <unistd.h>
+  #define GetCurrentDir getcwd
+#endif
+
+std::string FileUtil::GetCurrentWorkingDir() {
+  char buff[FILENAME_MAX];

```

Comment:
I think this may need to be modified slightly - if you run the program from a different folder, `getcwd` will give you the name of the folder you ran it from, not the name of the folder the program is actually in.  I'm not sure yet what the best thing is to do here, but we can discuss it more in the group chat.

Reply:
All right, I have (hopefully) updated it to find and use the filepath that you suggested.